app/controllers/two_factor_authentication/backup_code_verification_controller.rb

```diff
@@ -77,5 +77,9 @@ def handle_valid_backup_code
       redirect_to after_otp_verification_confirmation_url
       reset_otp_session_data
     end
+
+    def check_sp_required_mfa
+      check_sp_required_mfa_bypass(auth_method: 'backup_code')
```

Comment:
maybe for a future PR, but should we consider moving these strings to constants? minimize chance of typo later on?

Reply:
yeah, we definitely should